pkg/runtime/styleparam.go

```diff
@@ -149,7 +151,11 @@ func marshalTimeValue(value interface{}) (string, bool) {
 
 func styleStruct(style string, explode bool, paramName string, value interface{}) (string, error) {
 	if timeVal, ok := marshalTimeValue(value); ok {
-		return stylePrimitive(style, explode, paramName, timeVal)
+		styledVal, err := stylePrimitive(style, explode, paramName, url.QueryEscape(timeVal))
+		if err != nil {
+			return "", errorsw.Wrap(err, "failed to style time")
```

Comment:
I think here you cold just use `fmt.Errorf("failed to style time: %w", err)` instead and avoid the dependency on the errors pkg.

Reply:
We've already got the pkg/errors dependency in other places, it's a great package. Instead of importing it and errors, please just import pkg/errors which is compatible with errors, and using errors.Wrap() is a great idea. I'm going to clean up and use that e verywhere.